blank_project/package.json

```diff
@@ -12,14 +12,14 @@
     "test": "npm run build && jest test --env=near-shell/test_environment"
   },
   "devDependencies": {
-    "assemblyscript-json": "github:nearprotocol/assemblyscript-json",
+    "assemblyscript-json": "github:WebAssemblyOS/assemblyscript-json#near_dist",
```

Comment:
Is the plan going forward to switch to this branch instead of nearprotocol fork?

Reply:
No this was just because I was waiting on the PR since it fixes its exports.